src/Core/src/Handlers/Editor/EditorHandler.Android.cs

```diff
@@ -1,20 +1,16 @@
 ﻿using System;
 using Android.Views;
 using Android.Views.InputMethods;
-using AndroidX.AppCompat.Widget;
 using Microsoft.Maui.Graphics;
 using static Android.Views.View;
 
 namespace Microsoft.Maui.Handlers
 {
-	// TODO: NET8 issoto - Change the TPlatformView generic type to MauiAppCompatEditText
-	// This type adds support to the SelectionChanged event
-	public partial class EditorHandler : ViewHandler<IEditor, AppCompatEditText>
+	public partial class EditorHandler : ViewHandler<IEditor, MauiAppCompatEditText>
```

Comment:
We can't change the generic type.

I think the only change here we can look at doing is making `MauiAppCompatEditText` public 

I realize that makes the handler a little confusing :-/ 

But it'll be too breaking to change the generic type